### ISSUE TITLE
Showing new listings for Friday, 25 October 2024

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#48]
## Hybrid Quantum-Classical Reinforcement Learning in Latent Observation Spaces
- **Keywords:** ['hybrid quantum']
 - **Authors:** Dániel T. R. Nagy, Csaba Czabán, Bence Bakó, Péter Hága, Zsófia Kallus, Zoltán Zimborás
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.18284
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.18284
 - **Abstract**
 Recent progress in quantum machine learning has sparked interest in using quantum methods to tackle classical control problems via quantum reinforcement learning. However, the classical reinforcement learning environments often scale to high dimensional problem spaces, which represents a challenge for the limited and costly resources available for quantum agent implementations. We propose to solve this dimensionality challenge by a classical autoencoder and a quantum agent together, where a compressed representation of observations is jointly learned in a hybrid training loop. The latent representation of such an autoencoder will serve as a tailored observation space best suited for both the control problem and the QPU architecture, aligning with the agent's requirements. A series of numerical experiments are designed for a performance analysis of the latent-space learning method. Results are presented for different control problems and for both photonic (continuous-variable) and qubit-based agents, to show how the QNN learning process is improved by the joint training.
## Adiabatic training for Variational Quantum Algorithms
- **Keywords:** ['hybrid quantum']
 - **Authors:** Ernesto Acosta, Carlos Cano Gutierrez, Guillermo Botella, Roberto Campos
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Emerging Technologies (cs.ET)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.18618
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.18618
 - **Abstract**
 This paper presents a new hybrid Quantum Machine Learning (QML) model composed of three elements: a classical computer in charge of the data preparation and interpretation; a Gate-based Quantum Computer running the Variational Quantum Algorithm (VQA) representing the Quantum Neural Network (QNN); and an adiabatic Quantum Computer where the optimization function is executed to find the best parameters for the VQA. As of the moment of this writing, the majority of QNNs are being trained using gradient-based classical optimizers having to deal with the barren-plateau effect. Some gradient-free classical approaches such as Evolutionary Algorithms have also been proposed to overcome this effect. To the knowledge of the authors, adiabatic quantum models have not been used to train VQAs. The paper compares the results of gradient-based classical algorithms against adiabatic optimizers showing the feasibility of integration for gate-based and adiabatic quantum computing models, opening the door to modern hybrid QML approaches for High Performance Computing.
## Hybrid Quantum-Classical Feature Extraction approach for Image Classification using Autoencoders and Quantum SVMs
- **Keywords:** ['hybrid quantum']
 - **Authors:** Donovan Slabbert, Francesco Petruccione
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.18814
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.18814
 - **Abstract**
 In order to leverage quantum computers for machine learning tasks such as image classification, careful consideration is required: NISQ-era quantum computers have limitations, which include noise, scalability, read-in and read-out times, and gate operation times. Therefore, strategies should be devised to mitigate the impact that complex datasets can have on the overall efficiency of a quantum machine learning pipeline. This may otherwise lead to excessive resource demands or increased noise. We apply a classical feature extraction method using a ResNet10-inspired convolutional autoencoder to both reduce the dimensionality of the dataset and extract abstract and meaningful features before feeding them into a quantum machine learning block. The quantum block of choice is a quantum-enhanced support vector machine (QSVM), as support vector machines typically do not require large sample sizes to identify patterns in data and have short-depth quantum circuits, which limits the impact of noise. The autoencoder is trained to extract meaningful features through image reconstruction, aiming to minimize the mean squared error across a training set. Three image datasets are used to illustrate the pipeline: HTRU-1, MNIST, and CIFAR-10. We also include a quantum-enhanced one-class support vector machine (QOCSVM) for the highly unbalanced HTRU-1 set, as well as classical machine learning results to serve as a benchmark. Finally, the HTRU-2 dataset is also included to serve as a benchmark for a dataset with well-correlated features. The autoencoder achieved near-perfect reconstruction and high classification accuracy for MNIST, while CIFAR-10 showed poorer performance due to image complexity, and HTRU-1 struggled because of dataset imbalance. This highlights the need for a balance between dimensionality reduction through classical feature extraction and prediction performance using quantum methods.
## Characterizing conical intersections of nucleobases on quantum computers
- **Keywords:** ['hybrid quantum']
 - **Authors:** Yuchen Wang, Cameron Cianci, Irma Avdic, Rishab Dutta, Samuel Warren, Brandon Allen, Nam P. Vu, Lea F. Santos, Victor S. Batista, David A. Mazziotti
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Chemical Physics (physics.chem-ph); Computational Physics (physics.comp-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2410.18910
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2410.18910
 - **Abstract**
 Hybrid quantum-classical computing algorithms offer significant potential for accelerating the calculation of the electronic structure of strongly correlated molecules. In this work, we present the first quantum simulation of conical intersections (CIs) in a biomolecule, cytosine, using a superconducting quantum computer. We apply the Contracted Quantum Eigensolver (CQE) -- with comparisons to conventional Variational Quantum Deflation (VQD) -- to compute the near-degenerate ground and excited states associated with the conical intersection, a key feature governing the photostability of DNA and RNA. The CQE is based on an exact ansatz for many-electron molecules in the absence of noise -- a critically important property for resolving strongly correlated states at CIs. Both methods demonstrate promising accuracy when compared with exact diagonalization, even on noisy intermediate-scale quantum computers, highlighting their potential for advancing the understanding of photochemical and photobiological processes. The ability to simulate these intersections is critical for advancing our knowledge of biological processes like DNA repair and mutation, with potential implications for molecular biology and medical research.
